tools/memory

```diff
@@ -0,0 +1 @@
+../third_party/v8/tools/memory/
```

Comment:
From now on directory symlinks ought to be marked as such in //.gitattributes.
When git is upgraded on appveyor I'm going to remove the symlink hacks from third_party.py.

Reply:
Fixed.